test/bigtest/interactors/circulation-rules-editor/hints-menu.js

```diff
@@ -0,0 +1,38 @@
+import {
+  interactor,
+  isPresent,
+  action,
+  collection,
+  Interactor,
+  count,
+} from '@bigtest/interactor';
+
+import HintsSection from './hints-section';
+
+@interactor class HintsMenu {
+  static defaultScope = '.CodeMirror-hints';
+
+  arePresent = isPresent('.CodeMirror-hint');
```

Comment:
```suggestion
  isPresent = isPresent('.CodeMirror-hint');
```
I am thinking about `isPresent` since menu is sungular, what do you think?

Reply:
@VictorSoroka1  '.CodeMirror-hint' is applied to the every line of the hints menu, so I think it will be better to rename it to 'hasItems' (for example). Is it ok for you?

Reply:
Of course, thank you @YevheniiMaltsev.